account_statement_import_txt_xlsx/models/account_statement_import_sheet_mapping.py

```diff
@@ -69,6 +69,14 @@ class AccountStatementImportSheetMapping(models.Model):
         required=True,
         help="Amount of transaction in journal's currency",
     )
+    amount2_column = fields.Char(
```

Comment:
I'd use debit_amount and credit_amount and used absolute value of those instead.

Reply:
That is by far easier to understand w/o looking into code what `amount2` means IMHO

Reply:
Not really, as that means 3 extra fields (instead of 2) or less options (not being able to reverse the value if needed). Absolute values will not cover both options.

Reply:
@pedrobaeza what would be the point of reversing debit column value? If it's debit, just process its absolute value as debit. Reversing option is not needed at all

Reply:
Well, I'm not sure all cases fit such logic, but if you think so, let's go with it.

Reply:
Well, unless we can't think of any cases that would fail - I guess it's worth a try. `amount`, `debit_amount`, `credit_amount`. `amount` for compatibility looks at sign and believes it, `debit_amount` ignores sign and treats as debit, `credit_amount` similarly. The only case I can think of that would require `reverse_amount` boolean field is when there's only one column and it's reversed.

Reply:
Just don't do vague `amount2` please